string lighting_name
uint8 r
uint8 g
uint8 b
std_msgs/Float32 brightness # sets the brightness in range of 0..1 (0%..100%)